### ISSUE TITLE
Android devices fail to connect

### ISSUE THREAD

**Issue by muustafacokyasar** [AdguardTeam/AdGuardHome#7102]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, AMD64 (aka x86_64)

### Installation

Docker

### Setup

On one machine

### AdGuard Home version

0.107.51

### Action

-

### Expected result

-

### Actual result

-

### Additional information and/or screenshots

Windows and Apple devices can easily connect to the server, but private DNS connections cannot be established with Android devices. I've been using it for months and there was no problem, then suddenly it started to happen. Maybe it's because of the update.

**Comment by igorcv88** [comment]
Yeah this is truly frustrating... I've tried everything to get it working, like adding upstream DNS servers, editing android wifi parameters, reseting the router, generating new certificates for the domain... :/

**Comment by tjharman** [comment]
This is not an issue with AdGuardHome, but rather an issue with the SSL certificate you're using.  I struggled with this for a long time too.  [This explains](https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/) the issue.  Android seems to be much more picky about rejecting connections with the expired root in them.

To fix the issue I started using a different ACME SSL provider, not LetsEncrypt.

**Comment by tjharman** [comment]
It's also discussed in more details [here](https://community.letsencrypt.org/t/android-devices-with-dot-configured-interaction-with-new-default-chain/161020/14).
